projects/kit/components/slider/slider.component.ts

```diff
@@ -64,10 +64,26 @@ export class TuiSliderComponent {
         return Number(this.el.min);
     }
 
+    public set min(x: number) {
+        this.el.min = String(x);
+    }
+
     public get max(): number {
         return Number(this.el.max || 100);
     }
 
+    public set max(x: number) {
+        this.el.max = String(x);
+    }
+
+    public get step(): number {
+        return Number(this.el.step) || 1;
+    }
+
+    public set step(x: number) {
```

Comment:
Maybe it makes sense to add those as inputs + host binding instead of getter/setter couples relying on native props?